<launch>

<!-- Start Convex Hull, Plane Fitting, GUI, Waypoint Generator, and Move Group Interface nodes -->
<node pkg="uv_project" type="execute_basic_coverage.py" name="execute_basic_coverage" output="screen"/>
<node pkg="uv_project" type="irradiance_vectors.py"     name="irradiance_vectors"     output="screen"/>
<!-- <node pkg="fetch_sim2real" type="accumulation_map.py"       name="accumulation_map"       output="screen"/>
<node pkg="fetch_sim2real" type="sync_data.py"              name="sync_data"              output="screen"/> -->

</launch>
